will-kitchell/test/greet_test.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+var expect = require('chai').expect;
+var greet = require(__dirname + '/../lib/greet'); //must go back a directory, go into the lib directory, run greet.js
+
+describe('the greet function', function(){
+  it('should greet the user by name', function(){
+    expect(greet(process.argv[2])).to.eql('hello ' + process.argv[2]);
```

Comment:
As shown in Wednesday's lecture, you need to assign your own values to process.argv in order to test. Your unit tests need to be able to run by themselves and not rely on input.